pydantic/main.py

```diff
@@ -917,6 +921,7 @@ def create_model(
     __base__: Union[None, Type['Model'], Tuple[Type['Model'], ...]] = None,
     __module__: str = __name__,
     __validators__: Dict[str, 'AnyClassMethod'] = None,
+    __cls_kwargs__: Dict[str, Any] = None,
```

Comment:
@uriyyo I think this fix is should have been released as 2.0 as interface is changing and 1.9.1 no matter it's a default value because there are **field_definitions after this __cls_kwargs__ . Mypy was complaining and I had to add __cls_kwargs__={}, when calling it

Reply:
please create a separate issue.